src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -118,6 +118,26 @@
         }
     }
 
+    /**
+     * Retrieves POD addresses for all services in the specified {@code namespace} filtered by {@code podLabel}
+     * and {@code podLabelValue}.
+     *
+     * @param podLabel      label used to filter responses
+     * @param podLabelValue label value used to filter responses
+     * @return all POD addresses from the specified {@code namespace} filtered by the label
+     * @see <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#list-143">Kubernetes Endpoint API</a>
+     */
+    List<Endpoint> endpointsByPodLabel(String podLabel, String podLabelValue) {
```

Comment:
Could you at the same time rename `endpointsByLabel()` => `endpointsByServiceLabel()`?

Reply:
Done 👍